shop/src/main/java/ro/msg/learning/shop/Controller/LocationController.java

```diff
@@ -0,0 +1,27 @@
+package ro.msg.learning.shop.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.msg.learning.shop.DTO.LocationSimpleDto;
+import ro.msg.learning.shop.DTO.ProductAndProductCategoryDto;
+import ro.msg.learning.shop.Service.Implementation.LocationService;
+import ro.msg.learning.shop.Service.Implementation.ProductService;
+
+@RequestMapping("/location")
```

Comment:
Read a little about the best naming conventions for endpoints.
Reference: https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/